src/main/java/application/Routes.java

```diff
@@ -0,0 +1,37 @@
+package main.java.application;
+
+import main.java.server.Router;
+
+import java.util.HashMap;
+import java.util.TreeMap;
+
+public class Routes {
+    private static TreeMap<String, HashMap<String, Handler>> ROUTES;
+    static {
+        Router router = new Router();
+        router.get("/simple_get", Handlers.SimpleGet);
+        router.head("/simple_get", Handlers.SimpleGet);
```

Comment:
Interesting that every `router.head` route uses the `SimpleGet` handler 🤔 

Maybe just a quirk of the `router.get` responses we're delivering at the moment?